### DISCUSSION TITLE
Visual Picker - Design and Spec RFC

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3329] [Review request]
Hi all,

Paste is adding a Visual Picker component and we'd like your feedback.

![Screenshot 2023-07-14 at 9 48 47 AM](https://github.com/twilio-labs/paste/assets/87141551/ef749e54-a800-4d86-99a9-17067dd2719a)

### Component spec
Review [the component spec](https://docs.google.com/document/d/1RLuj07u04AdZoc_R3SmFTtmg6MjivtbuKNbNgtxPeKk/edit#heading=h.gwj72pv8f47o) any leave any comments/suggestions/questions directly in the google doc.

### Design
Review the below design proposal and leave any comments in this thread or in the [Figma file](https://www.figma.com/file/FjXhV7DHjXS1B3PmZxVwVf/Visual-Picker?type=design&node-id=169-112843&mode=design).
![Screenshot 2023-07-14 at 9 50 13 AM](https://github.com/twilio-labs/paste/assets/87141551/80581fc7-3029-4c4f-99e0-1d3902552330)

Thanks!

**Comment by TheSisb** [comment]
The component has been released and the window for comment has been closed.
